Xamarin.Forms.Platform.Android/Platform.cs

```diff
@@ -338,18 +338,8 @@ public static IVisualElementRenderer CreateRenderer(VisualElement element)
 
 		internal static IVisualElementRenderer CreateRenderer(VisualElement element, Context context)
 		{
-			IVisualElementRenderer renderer = null;
-
-			if (element is TemplatedView tv && tv.ResolveControlTemplate() != null)
-			{
-				renderer = new DefaultRenderer(context);
-			}
-
-			if (renderer == null)
-			{
-				renderer = Registrar.Registered.GetHandlerForObject<IVisualElementRenderer>(element, context)
-					?? new DefaultRenderer(context);
-			}
+			IVisualElementRenderer renderer = Registrar.Registered.GetHandlerForObject<IVisualElementRenderer>(element, context)
```

Comment:
This changed when added the Templated RadioButton. In the case of `TemplatedView`, `DefaultRenderer` was always used. I understand that the change was performance related. However, if the TemplatedView has an associated renderer registered it would stop working. @hartez  Check the PR when you can ;)